src/app/core/services/issue.service.ts

```diff
@@ -119,7 +121,9 @@ export class IssueService {
 
   createIssue(title: string, description: string, severity: string, type: string): Observable<Issue> {
     const labelsArray = [this.createLabel('severity', severity), this.createLabel('type', type)];
-    const hiddenData = new Map([['session', this.sessionId]]);
+    const appVersion = new ApplicationService(this.githubService).currentVersion;
```

Comment:
I just noticed this but in the future, we shouldn't need to construct the `ApplicationService` just to get the current version of the application. In fact, it should be a `static` variable as much as possible. Maybe we can bring this discussion outside of this PR after this PR has been merged!

Reply:
This is also partially due to the fact that I would need to do the same way as you did for #657 ><

Reply:
We can export `currentVersion` independently from `ApplicationService`. Can make the change in this PR.

Reply:
Can create a `currentVersion` variable that's export from `ApplicationService`'s module and used here.

Reply:
Yea, this sounds good! :+1: